src/contexts/BuySell/BuySellProvider.tsx

```diff
@@ -1,38 +1,28 @@
+import Web3 from 'web3'
 import React, { useState, useEffect, useCallback } from 'react'
 import { provider } from 'web3-core'
 
 import BigNumber from 'utils/bignumber'
 import BuySellContext from './BuySellContext'
-import { fetchTokenBuySellData } from 'utils/tokensetsApi'
 import useWallet from 'hooks/useWallet'
 import useBalances from 'hooks/useBalances'
 import useTransactionWatcher from 'hooks/useTransactionWatcher'
-import { useDebounce } from 'hooks/useDebounce'
-import { getUniswapTradeTransaction } from 'uniswap-sdk/uniswap'
-import {
-  getUniswapTradeType,
-  getUniswapCallData,
-  getUniswapTransactionOptions,
-} from './utils'
+import { getZeroExTradeData } from '../../utils/zeroExUtils'
```

Comment:
just `utils/zeroExUtils` should work here, similar to the bignumber import above